Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:latest
```

Comment:
We don't use latest tags because we want building our images to be reproducible. We should use the `3.8` tag.